SPECS/polkit/CVE-2025-7519.patch

```diff
@@ -0,0 +1,31 @@
+From 34ccf6c4d7a71872c9a216fde20dedb318a40e9a Mon Sep 17 00:00:00 2001
+From: Azure Linux Security Servicing Account
+ <azurelinux-security@microsoft.com>
+Date: Thu, 17 Jul 2025 06:46:54 +0000
+Subject: [PATCH] Fix CVE CVE-2025-7519 in polkit
+
+Upstream Patch Reference: https://github.com/polkit-org/polkit/commit/107d3801361b9f9084f78710178e683391f1d245.patch
+---
+ src/polkitbackend/polkitbackendactionpool.c | 6 ++++++
+ 1 file changed, 6 insertions(+)
+
+diff --git a/src/polkitbackend/polkitbackendactionpool.c b/src/polkitbackend/polkitbackendactionpool.c
+index 3894fe9..c9fa23e 100644
+--- a/src/polkitbackend/polkitbackendactionpool.c
++++ b/src/polkitbackend/polkitbackendactionpool.c
+@@ -672,6 +672,12 @@ _start (void *data, const char *el, const char **attr)
```

Comment:
@0xba1a The patch applies cleanly and the only delta is line numbers from the reference patch.